### PR TITLE
Tax Loss Harvest Alert V1

### PR DESCRIPTION
**Pull request by jglassenberg**
Adding the Tax Loss Harvest Alert sample

### CODE REVIEW

**Comment by jglassenberg** [comment]
@googlebot I signed it!

File: Tax Loss Harvest Alert/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+material_icon: event
+create_time: 2020-06-14
+update_time: 2020-006-14
+---
+
+Contributed by Jeremy Glassenberg [Site](https://www.apistrategist.com)
+
+When we purchase a stock and find the value dropped, there's an opportunity to sell that stock, purchase another, and claim a tax deduction.  However, many brokerages don't let you know when you have that opportunity.  
+
+This simple script works with Google Sheets to alert you for free.  Simply copy in your portfolio information, so the sheet knows what stocks you own and at what purchase prices. If anything falls below purchase price, you'll be notified of a tax loss harvest opportunity.
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/README.md:14]: Suggest rewording a little. We typically refrain from describing things as simple since it's vague (it might be simple for you, but not for somebody new to coding) unless it's truly trivial. Likewise, 'free' can also be problematic -- can be hidden or assumed 3P costs (e.g. ISP/bandwidth) or future changes in pricing that affect that.

File: Tax Loss Harvest Alert/src/code.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    if (row[6] < 0) {
+      message +=
+        row[1] +
+        ": " +
+        (parseFloat(row[6].toString()) * 100).toFixed(2).toString() +
+        "%<br>";
+      send_message = true;
+    }
+  }
+  //MailApp.sendEmail("jglassenberg@gmail.com","Tax-loss harvest", message);
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/src/code.js:47]: Might want to remove that comment :)

File: Tax Loss Harvest Alert/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## Technology highlights
+
+- The [`SpreadsheetApp` service][spreadsheetapp-docs] is used to provide stored information to the Apps Script
+- [Google Finance formula in Google Sheets](https://support.google.com/docs/answer/3093281?hl=en) are used to check stock prices in real-time and alert the script of changes
+
+
+## Try it
+
+1. Copy this [sheet][sheet-copy]
+2. In the calculations tab make sure your information in colums A through D are up to date
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/README.md:27]: colums -> columns

File: Tax Loss Harvest Alert/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+3. Copy and apply this [script] [script-copy] to check for tax-loss harvesting opportunities.  [TODO: confirm if this is needed]
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/README.md:28]: Resolve the TODO before we merge.

File: Tax Loss Harvest Alert/src/code.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+/*
+Whenever the spreadsheet changes,
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/src/code.js:2]: Comment and signature is a little confusing for me. Sounds like you're describing this like an onChange or onEdit trigger, but it's not, and the readme suggests setting it up as an time-based trigger.

File: Tax Loss Harvest Alert/src/code.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  }
+  //MailApp.sendEmail("jglassenberg@gmail.com","Tax-loss harvest", message);
+  if (!send_message) return;
+
+  MailApp.sendEmail({
+    to: SpreadsheetApp.getActiveSpreadsheet().getOwner().getEmail(),
+    subject: "Tax-loss harvest",
+    htmlBody: message,
+    /*inlineImages:
```
Comment (sqrrrl) [inline Tax Loss Harvest Alert/src/code.js:54]: Delete if not using.

File: Tax Loss Harvest Alert/src/code.js (hunk 2 above)
Comment (jglassenberg) [reply Tax Loss Harvest Alert/src/code.js:47]: Oh, good catch. I might need to be careful if that shows up in the github history.  Perhaps I should create a new branch and re-create this file.

File: Tax Loss Harvest Alert/src/code.js (hunk 5 above)
Comment (jglassenberg) [reply Tax Loss Harvest Alert/src/code.js:2]: It's actually triggered when there's a change to the spreadsheet - but that happens whenever portfolio stock prices change.  I'll see about clarifying it in the code comment and readme.
